serverless.yml

```diff
@@ -37,6 +41,8 @@ functions:
       DB_NAME: ${self:custom.db.connectInfo.DATABASE_NAME}
       DB_PASSWORD: ${self:custom.db.connectInfo.SCHEMA_OWNER_PASSWORD}
       AWS_DEPLOYMENT_REGION: ${self:provider.region}
+      BUCKET_NAME: iow-retriever-capture-${self:custom.environments.${self:provider.stage}}
```

Comment:
I'm don't think this necessary. The bucket name already comes in as part of the event that this lambda is invoked with: https://github.com/usgs/aqts-retriever-capture-raw-load/blob/master/src/etl/event_processor.py#L40

Reply:
done.